src/postgres-util/src/tunnel.rs

```diff
@@ -161,7 +161,7 @@ impl Config {
         task_name: &str,
         ssh_tunnel_manager: &SshTunnelManager,
     ) -> Result<Client, PostgresError> {
-        self.connect_traced(task_name, ssh_tunnel_manager, |_| ())
+        self.connect_traced(task_name, |_| (), ssh_tunnel_manager)
```

Comment:
Why did the ordering here get flipped back around?

Reply:
It is now consistently the last arg